### DISCUSSION TITLE
accessing files on linux (btrfs) seems impossible?

### DISCUSSION THREAD

**Discussion by Sinntax44** [ppy/osu#29741] [Q&A] [answered]
im using EndeavourOS xfce and when i go to the directory in which you can edit skins or where your can see your osu files i get this :
![image](https://github.com/user-attachments/assets/23060397-2e39-44bb-be62-e99aea4e577d)
![image](https://github.com/user-attachments/assets/2b84195f-2708-4f97-94bb-f3519b9c03bb)
![image](https://github.com/user-attachments/assets/b14f62f0-8f67-4f2a-8622-38c6c59b0f60)

everything is a jumbled mess. i don't know if it's a filesystem issue (using btrfs instead of the usual ext4)
any help/insight is appreciated!
i am using the appimage version
P.S i just wanna edit my skins man :(

**Comment by Joehuu** [comment, marked as answer]
There is a readme that explains why it's organised like that.

For now, you can use the export button, change `.osk` to `.zip`, unzip, edit, zip and rename back to `.osk`.

**Reply by bdach** [reply]
[wiki page in question](https://github.com/ppy/osu/wiki/User-file-storage)

**Reply by Sinntax44** [reply]
thank you, i didn't quite understand the readme.txt so this helps

**Reply by Sinntax44** [reply]
> There is a readme that explains why it's organised like that.
> 
> For now, you can use the export button, change `.osk` to `.zip`, unzip, edit, zip and rename back to `.osk`.

let's say i have whitecat's skin, so i export to osk, change it to zip, edit, zip, then rename to osk. if i import it back to lazer, will the client have another instance of the skin or will it override the existing one on the skins list?

**Reply by Joehuu** [reply]
Generally, it should overwrite the skin if the name of the `.osk` is the same and the name in the `skin.ini` is the same as an existing skin.

To be sure you have another instance of the skin, you can rename the `.osk`.
